bin/p2-dsctl/main.go

```diff
@@ -55,11 +59,19 @@ var (
 
 	cmdUpdate       = kingpin.Command(CmdUpdate, "Update a daemon set.")
 	updateID        = cmdUpdate.Arg("id", "The uuid for the daemon set").Required().String()
-	updateAZ        = cmdUpdate.Flag("az", "The availability zone of the pod cluster").String()
+	updateSelector  = cmdUpdate.Flag("selector", "The node selector, uses the same syntax as the test-selector command").Default(SelectorNotSpecified).String()
```

Comment:
@anthonybishopric: [Kingpin flags don't work that way](https://github.com/alecthomas/kingpin/blob/v2.2.2/values_generated.go#L77). `updateSelector` is a `*string`, but it will always be non-nil, regardless of whether the user passed the flag.